docs/pages/Getting_Started.md

```diff
@@ -2258,8 +2279,26 @@ obs_def modules (for example, `obs_def_1d_state_mod.f90` or
 files are included in the preprocessor namelist, a minimal `obs_def_mod.f90` is
 created which can only support identity forward observation operators.
 
-To add a new observation type which does not fit into any of the
-already-defined obs_def files, a new file should be created in the
+<span id="newQUANTITIES" class="anchor"></span> [](#newQUANTITIES)
+#### To add a new *quantity*,
```

Comment:
The style of #### Statement,

lowercase continuation of the statement

was confusing as I looked through this. Can you make this consistent with the rest of the document?

Reply:
Done.